### PR TITLE
Provide Dockerfile files for both JVM and Native modes

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #1118

This is also sort of necessary if an [AP4K](https://github.com/jbossas/protean-shamrock/pull/1031) guide is to make sense and not confuse users

### CODE REVIEW

**Review by cescoffier — CHANGES_REQUESTED** [review]
Once merged, the quickstarts need to be updated.

**Review by cescoffier — APPROVED** [review]

File: devtools/common/src/main/resources/templates/dockerfile-jvm.ftl [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/${project_artifactId}-jvm .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/${project_artifactId}-jvm
+#
+###
+FROM fabric8/java-jboss-openjdk8-jdk
```
Comment (gastaldi) [inline devtools/common/src/main/resources/templates/dockerfile-jvm.ftl]: I am not sure if this image is still maintained. Better use `jboss/base-jdk:8` instead?

Comment (geoand) [reply devtools/common/src/main/resources/templates/dockerfile-jvm.ftl]: @rhuss any idea?

**Comment by geoand** [comment]
Superseded by #1312

**Pull request closed without merging**